doc/source/serve/production.md

```diff
@@ -567,3 +557,216 @@ This section summarizes the best practices when deploying to production:
 * Use `serve config` to check the latest config that your Serve application received. This is its goal state.
 * Make lightweight configuration updates (e.g. `num_replicas` or `user_config` changes) by modifying your Serve config file and redeploying it with `serve deploy`.
 * Make heavyweight code updates (e.g. `runtime_env` changes) by starting a new Ray cluster, updating your Serve config file, and deploying the file with `serve deploy` to the new cluster. Once the new deployment is finished, switch your traffic to the new cluster.
+
+(serve-in-production-kubernetes)=
+
+## Deploying to Production on Kubernetes
+
+The above sections walk through how to deploy a Serve application manually using the Serve CLI.
+However, the best practice when running on Kubernetes is to use the [RayService] custom resource (CR) that's provided as part of [KubeRay] because it automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
+
+A [RayService] CR encapsulates a multi-node Ray Cluster and a Serve application that will run on top of it into a single Kubernetes manifest.
+Deploying, upgrading, and getting the status of the application can be done using standard `kubectl` commands.
+This section will walk through how to deploy, monitor, and upgrade the [`FruitStand` example](serve-in-production-example) from above on Kubernetes.
+
+### Installing the KubeRay operator
+
+The first step is to install the `KubeRay` operator into your Kubernetes cluster.
+This will create a pod that runs the `KubeRay` controller to manage resources based on the `RayService` CRs you create.
+
+Install the operator using `kubectl apply` and check that the controller pod is running:
+```console
+$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/default"
```

Comment:
`"github.com/ray-project/kuberay/ray-operator/config/default"` leads to a 404 page when I enter it in the browser. Should it lead to a specific page?

Should we include a commit hash here to permalink to a specific operator? Is there a chance that the operator could get updated and break this example?

Reply:
@shrekris-anyscale this is the pattern used by kubectl to clone the repo which is is fine...

However, this command isn't even on Kuberay docs 
https://ray-project.github.io/kuberay/deploy/installation/

I don't think it will work out of the box for users?

Reply:
fixed